server/plugin/webhook.go

```diff
@@ -61,6 +74,17 @@ func signBody(secret, body []byte) ([]byte, error) {
 	return computed.Sum(nil), nil
 }
 
+// GetEventWithRenderConfig wraps any github Event into an EventWithRenderConfig
+// which also contains per-subscription configuration options.
+func GetEventWithRenderConfig(event interface{}, sub *Subscription) *EventWithRenderConfig {
+	return &EventWithRenderConfig{
+		Event: event,
```

Comment:
Do we lose any static typing from resorting to `interface{}`? I think not because these are just being passed into templates, so the event is not referenced in statically-typed Go code anywhere.